src/Components/Feats/index.js

```diff
@@ -519,30 +519,51 @@ class Feats extends React.Component {
         this.state = this.feats[0];
 
         this.handleMethodChange = event => {
-            const feat = this.feats.filter(feat => feat.id == event.target.value)
+            let feat = this.feats.filter(feat => feat.id == event.target.value)
```

Comment:
I believe this should be a const since it's instantiated each time `this.handleMethodChange` is invoked, and it isn't modified inside it's lexical scope (not sure if that's the right term) -- same thing would apply to the `focusedFeat` var in `render()`

Reply:
Yes, lexical scope is the correct wording to use here.  I'll revert this change.